docs/install/install-visual-studio.md

```diff
@@ -93,6 +93,11 @@ To install Visual Studio 2017 RC in a language of your choosing, click the **Lan
 
   > [!IMPORTANT]
   > While Visual Studio 2017 RC in general is supported for use in a production environment, those workloads and components that are marked "Preview" in the installation UI are not supported for use in a production environment.
+  
+## Changing the installer Language
```

Comment:
This should also be a level 3 heading, as it is a child of the parent, "Install language packs"
Also, please change this to "Change the installer language," because it's a best practice to avoid gerunds (words ending in '-ing' whenever we can.